src/Collection/Collection.model.ts

```diff
@@ -25,6 +26,18 @@ export class Collection extends Model<CollectionAttributes> {
         WHERE contract_address = ANY(${contractAddresses})`)
   }
 
+  static async findCollectionOwningItem(
```

Comment:
I like this name, but I think we were using ByXXX if you want to be consistent ( `findByItemId` )

Reply:
Changed it to `findByItemId`!